component/src/main/java/io/siddhi/extension/io/cdc/util/CDCSourceConstants.java

```diff
@@ -91,4 +91,12 @@
     public static final String MONGO_SET = "$set";
     public static final String MONGO_OBJECT_NUMBER_LONG = "$numberLong";
     public static final String MONGO_OBJECT_NUMBER_DECIMAL = "$numberDecimal";
+
+    // Constants for Cron Support
+    public static final String CRON_EXPRESSION = "cron.expression";
+    public static final String JOB_GROUP = "JobGroup";
```

Comment:
Shall we add some meaningful names if those are getting printed when debugging or on errors?? Eg: CDCPollingGroup

Reply:
fixed in ee838f87eb853dd4cccb33250c2fa8f7d620a3cc